src/actions/peers.js

```diff
@@ -31,14 +31,20 @@ const login = (dispatch, getState, data, config) => {
     const activePeer = store.peers.data ||
       new Lisk.APIClient(config.nodes, { nethash: config.nethash });
     const address = extractAddress(publicKey);
-    const accountBasics = {
+    let accountBasics = {
```

Comment:
you don't need `let` here, with spread operator you can add hwInfo in the initial definition of accountBasics